tests/Drupal/Commenting/DocCommentUnitTest.php

```diff
@@ -33,6 +33,8 @@ protected function getErrorList(string $testFile): array
                 66  => 1,
                 100 => 4,
                 101 => 1,
+                130 => 1,
```

Comment:
wrong line number, the error is on 131. Can we use the pointer to the tag instead?

Reply:
Good call, have changed this now.